### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()

### ISSUE THREAD

**Issue by flemeur** [s-panferov/awesome-typescript-loader#374]
I get this deprecation warning when running a build

```
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

See: https://github.com/webpack/loader-utils/issues/56

**Comment by fulls1z3** [comment]
I have exactly the same, all builds are failing on 3.0.4

**Comment by ZuBB** [comment]
> builds are failing on 3.0.4

😢

**Comment by s-panferov** [comment]
Fixed by awesome-typescript-loader@3.0.5, thanks to @sod
